### ISSUE TITLE
Record Command Improvement 

### ISSUE THREAD

**Issue by maxgestic** [minipunch/usarealismrp-fxserver#1183]
Maybe have it so you can search via Name and DOB so you can find players that are not online

**Comment by maxgestic** [comment]
@Savage912-Official

**Comment by Savage912-Official** [comment]
Yeah that was the plan I just forgot about it and started working on other things.

**Comment by minipunch** [comment]
Might as well just also put this in the MDT at that point cause it does all
that already lol

**Comment by maxgestic** [comment]
Yea showing all the vehicles above the criminal history would be very useful

**Comment by Savage912-Official** [comment]
Working on it now

**Comment by minipunch** [comment]
@Savage912-Official Update on this ?
